src/index.js

```diff
@@ -0,0 +1,13 @@
+import './style.css';
+
+function component() {
+    const element = document.createElement('div');
+
+    // Lodash, now imported by this script
+    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+    element.classList.add('hello');
+
+    return element;
+  }
+
+  document.body.appendChild(component());
```

Comment:
I guess you used this to test the webpack but I think it will look cleaner if they are deleted as they will not be needed for the next phase of the project 😀